packages/kit/test/apps/options/source/pages/+layout.svelte

```diff
@@ -1,7 +1,20 @@
 <script>
+	import { onMount } from 'svelte';
 	import { setup } from '../../../../setup.js';
 
 	setup();
+
+	// TODO: remove this when Svelte 5 addresses this issue https://github.com/sveltejs/svelte/issues/14438
+	onMount(() => {
+		// @ts-expect-error trustedTypes is a limited availability global
+		// see https://developer.mozilla.org/en-US/docs/Web/API/Window/trustedTypes
+		if (window.trustedTypes && trustedTypes.createPolicy) {
+			// @ts-expect-error
+			trustedTypes.createPolicy('default', {
+				createHTML: (/** @type {string} */ str) => str
+			});
+		}
+	});
```

Comment:
temporarily added this trust type policy fix until it's addressed by either Svelte or SvelteKit